doc/source/management/audit_log_plugin.rst

```diff
@@ -40,8 +40,12 @@ Example of the Disconnect event: ::
   "DB"=""
   />
 
-* **Query** - Additional fields for this event are: ``COMMAND_CLASS`` (values come from the ``com_status_vars`` array in the :file:`sql/mysqld.cc`` file in a MySQL source distribution. Examples are ``select``, ``alter_table", "create_table", etc.), ``CONNECTION_ID``, ``STATUS`` (indicates error when non-zero), ``SQLTEXT`` (text of SQL-statement), ``USER``, ``HOST``, ``OS_USER``, ``IP``. Possible values for the ``NAME`` name field for this event are ``Query``, ``Prepare``, ``Execute``, ``Change user``, etc.
+* **Query** - Additional fields for this event are: ``COMMAND_CLASS`` (values come from the ``com_status_vars`` array in the :file:`sql/mysqld.cc`` file in a MySQL source distribution. Examples are ``select``, ``alter_table``, ``create_table``, ``show_errors``, etc.), ``CONNECTION_ID``, ``STATUS`` (indicates error when non-zero), ``SQLTEXT`` (text of SQL-statement), ``USER``, ``HOST``, ``OS_USER``, ``IP``. Possible values for the ``NAME`` name field for this event are ``Query``, ``Prepare``, ``Execute``, ``Change user``, etc.
 
+.. note::
+ 
+   It may be not obvious, but ``COMMAND_CLASS="error"`` entry in the audit logs is triggered by ``show_error`` value, and just means that errors are allowed to be shown.
```

Comment:
What is the source for this statement?

Reply:
With the note removed, is there any content remaining in the PR?

Reply:
This PR also contains a pair of formatting/typo fixes